src/TabToGettextVisitor.php

```diff
@@ -133,4 +132,14 @@ private function checkNbSubstitutions(Node $node, string $sentence, int $expecte
             throw new MismatchSubstitutionCountException("Expected substitution count differs (expected: $expected_count, actual: $actual_count) for: «${sentence}» in: $this->filepath at L$line");
         }
     }
+
+    private function getSentenceToTranslate(Node $node, string $primary, string $secondary): string
+    {
+        try {
+            return $this->dictionary->get($primary, $secondary);
+        } catch (EntryNotFoundException $exception) {
+            $line = $node->getAttribute('startLine');
+            throw new SentenceNotFoundException("Cannot found ($primary, $secondary) in: $this->filepath at L$line");
```

Comment:
May I suggest to build this error message directly in the SentenceNotFoundException class? This would make sure future developments does not throw this exception without any information which would make hard to debug/fix anything.

Reply:
ok